workspace/notebook/pins_inspectors.json

```diff
@@ -0,0 +1,538 @@
+					"            )) <> T2.RowID  -- same row, changed data\r\n",
+					"        THEN 'Y'\r\n",
+					"        WHEN T2.horizonId IS NULL -- new data\r\n",
+					"        THEN 'Y'\r\n",
+					"    ELSE 'N'\r\n",
+					"    END  = 'Y')\r\n",
+					"    AND T1.horizonId IS NOT NULL\r\n",
+					"    AND T1.expected_from = (SELECT MAX(expected_from) FROM odw_standardised_db.pins_inspectors)\r\n",
+					";"
+				],
+				"execution_count": 13
+			},
+			{
+				"cell_type": "markdown",
+				"metadata": {
+					"nteract": {
+						"transient": {
+							"deleting": false
+						}
+					}
+				},
+				"source": [
+					"# Dataset is created that contains changed data and corresponding target data\n",
+					"- This script combines data that has been updated, Deleted or is new, with corresponding target data\n",
+					"- View **pins_inspectors_new** is unioned to the target data filter to only those rows where changes have been detected\n",
+					"## View pins_inspectors_changed_rows is created"
+				]
+			},
+			{
+				"cell_type": "code",
+				"metadata": {
+					"jupyter": {
+						"source_hidden": false,
+						"outputs_hidden": false
+					},
+					"nteract": {
+						"transient": {
+							"deleting": false
+						}
+					},
+					"microsoft": {
+						"language": "sparksql"
+					},
+					"collapsed": false
+				},
+				"source": [
+					"%%sql\r\n",
+					"-- Create new and updated dataset\r\n",
+					"\r\n",
+					"CREATE OR REPLACE TEMPORARY VIEW pins_inspectors_changed_rows\r\n",
+					"    AS\r\n",
+					"-- gets updated, deleted and new rows \r\n",
+					"Select \r\n",
+					"    InspectorsId,\r\n",
+					"    horizonId,\r\n",
+					"    firstName,\r\n",
+					"    lastName,\r\n",
+					"    postName,\r\n",
+					"    organisationName,\r\n",
+					"    title,\r\n",
+					"    salutation,\r\n",
+					"    qualifications,\r\n",
+					"    email,    \r\n",
+					"    Migrated,\r\n",
+					"    ODTSourceSystem,\r\n",
+					"    IngestionDate,\r\n",
+					"    ValidTo,\r\n",
+					"    RowID,\r\n",
+					"    IsActive\r\n",
+					"\r\n",
+					"From pins_inspectors_new WHERE HistoricIsActive = 'Y' or HistoricIsActive IS NULL\r\n",
+					"\r\n",
+					"    UNION ALL\r\n",
+					"\r\n",
+					"-- gets original versions of updated rows so we can update EndDate and set IsActive flag to 'N'\r\n",
+					"SELECT\r\n",
+					"    InspectorsId,\r\n",
+					"    horizonId,\r\n",
+					"    firstName,\r\n",
+					"    lastName,\r\n",
+					"    postName,\r\n",
+					"    organisationName,\r\n",
+					"    title,\r\n",
+					"    salutation,\r\n",
+					"    qualifications,\r\n",
+					"    email,\r\n",
+					"    Migrated,\r\n",
+					"    ODTSourceSystem,\r\n",
+					"    IngestionDate,\r\n",
+					"    ValidTo,\r\n",
+					"    RowID,\r\n",
+					"    IsActive\r\n",
+					"    \r\n",
+					"FROM odw_harmonised_db.pins_inspectors\r\n",
+					"WHERE horizonId IN (SELECT horizonId FROM pins_inspectors_new WHERE horizonId IS NULL) AND IsActive = 'Y'; "
```

Comment:
isn't this line always going to return everything? I don't think you need the where horizon Id is null clause